apps/trading/e2e/tests/referrals/test_referrals.py

```diff
@@ -0,0 +1,151 @@
+import pytest
+from playwright.sync_api import Page, expect
+from vega_sim.service import VegaService
+from conftest import init_vega
+from fixtures.market import setup_continuous_market, setup_simple_market
+from actions.utils import change_keys, create_and_faucet_wallet, forward_time, selector_contains_text
+from actions.vega import submit_order, submit_liquidity
+from wallet_config import MM_WALLET, PARTY_A, PARTY_B
+
+SELL_ORDERS = [[1, 111], [1, 111], [1, 112], [1, 112], [
+    1, 113], [1, 113], [1, 114], [1, 114], [1, 115], [1, 115]]
+BUY_ORDERS = [[1, 106], [1, 107], [1, 108]]
+
+
+@pytest.fixture(scope="module")
+def vega(request):
+    with init_vega(request) as vega:
+        yield vega
+
+
+@pytest.fixture(scope="module")
+def continuous_market(vega):
+    market = setup_simple_market(vega, custom_quantum=100000)
+    return setup_continuous_market(vega, market, BUY_ORDERS, SELL_ORDERS, add_liquidity=False)
+
+
+def generate_referrer_expected_value_dic(expected_base_commission, expected_staking_multiplier, expected_final_commission_rate, expected_volume, expected_num_traders, expected_total_commission):
+    return {
+        '[data-testid=my-volume-value]': expected_volume,
+        '[data-testid=total-commission-value]': expected_total_commission,
+        '[data-testid=base-commission-rate-value]': expected_base_commission,
+        '[data-testid=number-of-traders-value]': expected_num_traders,
+        '[data-testid=final-commission-rate-value]': expected_final_commission_rate,
+        '[data-testid=staking-multiplier-value]': expected_staking_multiplier
+    }
+
+
+def generate_referral_expected_value_dic(expected_volume, expected_tier, expected_discount, expected_epochs, expected_epochs_to_next_tier):
+    return {
+        '[data-testid=combined-volume-value]': expected_volume,
+        '[data-testid=current-tier-value]': expected_tier,
+        '[data-testid=discount-value]': expected_discount,
+        '[data-testid=epochs-in-set-value]': expected_epochs,
+        '[data-testid=epochs-to-next-tier-value]': expected_epochs_to_next_tier
+    }
+
+
+def check_tile_values(page: Page, expected_results: dict):
```

Comment:
We need to confirm it's "ok" to refresh the page in order to see the data, It might be an issue that we have to refresh. As a general rule in console the most up to date data should always be displayed ( no refresh required).

Doesn't block this pr though.

Reply:
This is as expected. Confirmed with @asiaznik